src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -554,18 +579,6 @@ public SharedTableBuilder withTablePrefix(String tablePrefix) {
         return this;
     }
 
```

Comment:
It seems like we should not expose setters for these two, since we would always ignore what the user could have set and override them in build().